keptn/keptn
# feat: Removed clean up of uniform and services when deleting a project

## This PR

Removed clean-up of services from:
- The CLI command keptn delete project 
- The Delete Project API endpoint

### Related Issues


Fixes #7297

### Notes

As explained in #8619 we completely removed the uniform clean up of services as well

### How to test

Run this version of shippy and CLI with keptn delete project, check that the git repo still has the services 
Integration test run @: https://github.com/keptn/keptn/actions/runs/2903390790
